sources/packages/web/src/services/http/StudentAppealApi.ts

```diff
@@ -24,12 +25,22 @@ export class StudentAppealApi extends HttpBaseClient {
     );
   }
 
+  /**
+   * Get student application appeal.
+   * @param appealId appeal id.
+   * @param studentId student id.
+   * @returns student application appeal.
+   */
   async getStudentAppealWithRequests(
     appealId: number,
-  ): Promise<StudentAppealAPIOutDTO> {
-    return this.getCall<StudentAppealAPIOutDTO>(
-      this.addClientRoot(`appeal/${appealId}/requests`),
-    );
+    studentId?: number,
+  ): Promise<StudentAppealAPIOutDTO | DetailedStudentAppealAPIOutDTO> {
```

Comment:
Just a suggestion, when there is some benefit we can apply the same for these methods also.
https://github.com/bcgov/SIMS/pull/1924#discussion_r1191476220